doc/using_statistics.md

```diff
@@ -134,22 +136,93 @@ Server: Jetty(9.1.5.v20140505)
 }
 ```
 
+**Connecting The Focus To The Publisher For PubSub Method**
+
+Upon start of the Jitsi Videobridge, it will send the following stanza to the focus
```

Comment:
The videobridge creates a pubsub node on the pubsub component and then publishes updates to that node. Then the focus subscribes to that pubsub node. The stats are not sent directly to the focus